mattermost/mattermost-plugin-legal-hold
# release_hold_test

#### Summary
**This test is to remove legal hold on legal hold page**

**An admin user can release (delete) a legal hold**
**Test Scenario**

- Go to legal hold page
- Check that there are legal holds on the page
- Then click the remove button to remove a legal hold
- confirm that hold is deleted successfully

#### Screenshot
<img width="1209" alt="Screenshot 2024-11-11 at 13 30 09" src="https://github.com/user-attachments/assets/f9302efd-f6e3-4730-9136-1cebd85525a8">
